docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+services:
+  whatsapp-chatgpt:
+    image: ghcr.io/askrella/whatsapp-chatgpt:latest
```

Comment:
i guess we need a pipeline to made this work

Reply:
Oh yes yes.

Reply:
would you be so kind and add a pipeline too?

Reply:
> would you be so kind and add a pipeline too?

Of course adding

Reply:
@EzxD  added kind sir